### PR TITLE
Porting blmc_robot solo12 to odri_control_interface

### PR DESCRIPTION
**Pull request by jviereck**
This is an updated version against the latest version of the odri control interface. I've tested it with the blmc_robots/solo12 demo on the real robot.

### CODE REVIEW

File: demos/demo_solo12_calibration.cpp [hunk 1]
```diff
@@ -1,60 +0,0 @@
-/**
```
Comment (MaximilienNaveau) [inline demos/demo_solo12_calibration.cpp:1]: Why removing this demo?

**Review by MaximilienNaveau — COMMENTED** [review]
Seems fine to me,
Why removing the calibration demo?
This would help people know how to calibrate solo12 using the api.

File: demos/demo_solo12_calibration.cpp (hunk 1 above)
Comment (MaximilienNaveau) [reply demos/demo_solo12_calibration.cpp:1]: I did a mistake merging the demos/CMakeLists.txt BTW there shouldn't be the demo_solo12_calibration as you deleted the file.
I can fix it but I wonder why?

**Comment by jviereck** [comment]
> Why removing the calibration demo?
> This would help people know how to calibrate solo12 using the api.

I felt the demo was not buying much more now that the other solo12 demo does the calibration as well. However, it brings a bit of overhead to port the demo. That's why I decided to delete it.

Do you want me to bring it back?

**Comment by MaximilienNaveau** [comment]
Ok no it makes sense.
You should delete the line in the demos/CMakeLists.txt that compiles it then.
Sorry for this.

**Comment by MaximilienNaveau** [comment]
I will fix the CMakeLists.txt as it seems there is a trick.
Every demo need the path to the yaml file then.

**Pull request merged**